packages/blockchain-wallet-v4-frontend/src/components/IdentityVerification/IdvIntro/index.tsx

```diff
@@ -0,0 +1,178 @@
+                id='scenes.verifyemail.title'
+                defaultMessage='Verify Your Email'
+              />
+            </Text>
+            <Text size='12px' lineHeight='150%' weight={500}>
+              <FormattedMessage
+                id='copy.swap_confirm'
+                defaultMessage='Confirm your email address to protect your Blockchain.com Wallet.'
+              />
+            </Text>
+          </div>
+        </FlexStartRow>
+        <FlexStartRow style={{ marginBottom: '28px' }}>
+          <IconBackground size='32px' style={{ marginRight: '16px' }}>
+            <Text color='blue600' size='20px' weight={600}>
+              2
+            </Text>
+          </IconBackground>
+          <div>
+            <Text color='grey900' size='14px' weight={600} lineHeight='150%'>
+              <FormattedMessage
+                id='copy.swap_add_name'
+                defaultMessage='Add Your Name and Address'
+              />
+            </Text>
+            <Text size='12px' lineHeight='150%' weight={500}>
+              <FormattedMessage
+                id='copy.swap_name_address'
+                defaultMessage='We need to know your name and address to comply with local laws.'
+              />
+            </Text>
+          </div>
+        </FlexStartRow>
+        <FlexStartRow style={{ marginBottom: '62px' }}>
+          <IconBackground size='32px' style={{ marginRight: '16px' }}>
+            <Text color='blue600' size='20px' weight={600}>
+              3
+            </Text>
+          </IconBackground>
+          <div>
+            <Text color='grey900' size='14px' weight={600} lineHeight='150%'>
+              {props.resultTitle}
+            </Text>
+            <Text size='12px' lineHeight='150%' weight={500}>
+              {props.resultCopy}
+            </Text>
+          </div>
+        </FlexStartRow>
+        <Button
+          nature='primary'
+          data-e2e='swapVerify'
+          height='48px'
+          onClick={() =>
+            props.idvActions.verifyIdentity(props.selectedTier, false)
+          }
+          fullwidth
+        >
+          <FormattedMessage
+            id='buttons.verify_now'
+            defaultMessage='Verify Now'
+          />
+        </Button>
+        {props.goBack && (
+          <BackLink
+            data-e2e='goBack'
+            onClick={props.goBack}
+            weight={500}
+            size={'14px'}
+          >
+            <FormattedMessage id='buttons.go_back' defaultMessage='Go Back' />
+          </BackLink>
+        )}
+      </FlyoutWrapper>
+    </>
+  )
+}
+
+const mapDispatchToProps = (dispatch: Dispatch) => ({
+  idvActions: bindActionCreators(
+    actions.components.identityVerification,
+    dispatch
+  )
+})
+const connector = connect(undefined, mapDispatchToProps)
+
+const enhance = compose(connector)
+
+type OwnProps = {
+  goBack?: () => void
+  handleClose: () => void
+  resultCopy: React.ReactElement<any, any>
```

Comment:
Will this pass the no explicit any check?

Reply:
I didn't change this code just moved everything to one component